mcs/class/System.Drawing/System.Drawing/gdipFunctions.cs

```diff
@@ -979,22 +988,22 @@ static internal void CheckStatus (Status status)
 		internal static extern Status GdipBitmapSetPixel (IntPtr bmp, int x, int y, int argb);
 
 		// Image functions
-		[DllImport(GdiPlus, CharSet=CharSet.Auto)]
+		[DllImport(GdiPlus, CharSet=GdiPlusCharSet)]
 		internal static extern Status GdipLoadImageFromFile ( [MarshalAs(UnmanagedType.LPWStr)] string filename, out IntPtr image );
```

Comment:
I wonder if we can't just use CharSet.Unicode in the Mono implementation as well here and in most other cases. It's marshalled as a Unicode/LPWStr anyway after all.